### PR TITLE
docs: add missing punctuation in document

### PR DESCRIPTION
**Pull request by lumirlumir**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Hello, I've added missing punctuation in `governance.md`.

There is always punctuation at the end of the lists (represented with `*` in Markdown), as shown in the screenshot below. However, the items I added were missing it.

image

#### Is there anything you'd like reviewers to focus on?

```markdown
* Technical direction
* Project governance and process (including this policy)
* Contribution policy
* GitHub repository hosting
```
The sentences in the list above are incomplete, so it’s possible to omit the periods. However, since periods are used consistently throughout the Markdown page, I’m debating whether to include them here as well.

### CODE REVIEW

**Review by Tanujkanti4441 — COMMENTED** [review]
Hi @lumirlumir, thanks for the PR!

There is inconsistency regarding `full stop` punctuation in docs pages like:

We have full stop here in list
Screenshot 2024-11-25 190356

but not here
Screenshot 2024-11-25 190333

And there are few more place where punctuation is not consistent in the same list such as:

in [manage-issues](https://eslint.org/docs/latest/maintain/manage-issues) page.
Screenshot 2024-11-25 190206

or in [work-on-issue](https://eslint.org/docs/latest/contribute/work-on-issue) page
Screenshot 2024-11-25 190139

i think we should also make it consistent on other pages.

**Comment by lumirlumir** [comment]
Hello, @Tanujkanti4441

Thank you for the detailed review!  

I've actually been checking for other missing punctuation in the documentation. If that's okay, would it be alright to add the missing punctuation to all the other document pages as well? I've noticed that many sentences in the lists are missing punctuation.  

So far, I've reviewed about a third of the documents and found missing punctuation in more than 10 pages. By the time I'm done, I estimate that there could be issues in over 20 pages.  

Now, I'm wondering how best to proceed with this task:  

1. Find all the missing punctuation, integrate them, and update this PR. However, this might make the PR too large to review.  
2. Split the process by category directories (e.g., "INTEGRATE ESLINT," "CONTRIBUTE TO ESLINT," etc.). 
3. ...Other suggestions...

It would be helpful to get your advice on this. I'll follow your recommendation!  

Once the direction is decided, I'll let you know when I expect to finish this work.

**Comment by Tanujkanti4441** [comment]
> Now, I'm wondering how best to proceed with this task:
> 
>     1. Find all the missing punctuation, integrate them, and update this PR. However, this might make the PR too large to review.
> 
>     2. Split the process by category directories (e.g., "INTEGRATE ESLINT," "CONTRIBUTE TO ESLINT," etc.).
> 
>     3. ...Other suggestions...

IMHO, it's better to update this PR because on the basis of what i have noticed in docs i am assuming even the number of pages would be large but the content to update (adding punctuation) won't be that much in per page.

But still let's ask other @eslint/eslint-team members that how they would like the PR to be made.

File: docs/src/contribute/governance.md [hunk 1]
```diff
@@ -111,10 +111,10 @@ The ESLint project is jointly governed by a Technical Steering Committee (TSC) w
...
 The TSC has final authority over this project including:
 
-* Technical direction
-* Project governance and process (including this policy)
-* Contribution policy
-* GitHub repository hosting
+* Technical direction.
+* Project governance and process (including this policy).
+* Contribution policy.
+* GitHub repository hosting.
```
Comment (nzakas) [inline docs/src/contribute/governance.md:117]: These should not have periods.

**Comment by nzakas** [comment]
I think it's fine to update this PR will all of the changes.

Just note that not all bullets required periods, only those that are complete sentences.

**Comment by lumirlumir** [comment]
Thanks, @Tanujkanti4441 and @nzakas.

I think I can finish this by the end of the week. Until then, I'll keep it as a draft.  

Once the process is complete, I'll ping the reviewers!

**Comment by lumirlumir** [comment]
Hello, @Tanujkanti4441 @nzakas,  

I've added as many missing punctuation marks as I could.  

I only added them to complete sentences, not to incomplete ones.  

Additionally, the `Yes.` in the screenshot below uses a `.` while the others don’t. For consistency, I deleted it.

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**